artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/jgroups/JChannelManager.java

```diff
@@ -34,7 +34,7 @@
 
    private static final Logger logger = Logger.getLogger(JChannelManager.class);
 
-   private Map<String, JChannelWrapper> channels;
```

Comment:
@clebertsuconic The intention was that multiple instances of JChannelManager would share the same set of channels. However, I see that I forgot to synchronize global access to this map, so this change has race conditions. 

In any case, what approach would you prefer to make wrt. making only one channel get created across broadcast and discovery groups?